src/openrct2/util/Util.cpp

```diff
@@ -172,12 +172,34 @@ int32_t bitscanforward(int32_t source)
     int32_t success = __builtin_ffs(source);
     return success - 1;
 #else
-#    pragma message "Falling back to iterative bitscan forward, consider using intrinsics"
+#    pragma message("Falling back to iterative bitscan forward, consider using intrinsics")
     // This is a low-hanging optimisation boost, check if your compiler offers
     // any intrinsic.
     // cf. https://github.com/OpenRCT2/OpenRCT2/pull/2093
     for (int32_t i = 0; i < 32; i++)
-        if (source & (1u << i))
+        if (source & (1ull << i))
```

Comment:
Is it necessary to also change the 32-bit version?

Reply:
It's not really necessary is because I thought i could get rid of the 32 bit version but realised it would prevent optimisations on x86-32

Reply:
Why are you casting to a 64-bit type in the 32-bit version?